### PR TITLE
Version bump `v0.5.15`

### PR DESCRIPTION
**Pull request by bharat-thotakura**

### CODE REVIEW

**Review by vtomole — APPROVED** [review]
LGTM

**Comment by richrines1** [comment]
#938 should fix tests

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -147,7 +147,7 @@ jobs:
           python-version: 3.8
       - name: Install dependencies, using minimum supported versions
         run: |
-          min_versions=$(sed ${{ matrix.type }}/*requirements.txt -e 's/[~>]=\([0-9]\)/==\1/')
+          min_versions=$(sed ${{ matrix.type }}/*requirements.txt -e 's/[~>]=\([0-9]\)/==\1/' -e '/checks-superstaq/d' -e '/general-superstaq/d' -e '/qiskit-superstaq/d' -e '/cirq-superstaq/d')
```
[the remark this answers is not in the document]
Comment (bharat-thotakura) [reply .github/workflows/ci.yml:150]: Yes! Thanks, that's neater

**Comment by richrines1** [comment]
fwiw it also seems to work if we use non-editable installs of the local packages - see #940

**Comment by bharat-thotakura** [comment]
I see, I think I'll go [with that then](https://github.com/Infleqtion/client-superstaq/pull/936/commits/7a3ef7fc97fcb2b5e30a82c33e9ac12dfedbc092)

**Review by richrines1 — APPROVED** [review]

**Pull request merged**